index.html

```diff
@@ -0,0 +1,170 @@
+                            Listen to lectures from speakers from around the world and 
+                            learn about the latest trends in the world.
+                        </p>
+                    </div>
+                </article>
+                <article class="prog program-2">
+                    <div class="program-icon-img transparency">
+                        <img src="images/program_icon_02.png" alt="CC Exhibition Icon">
+                    </div>
+                    <h3 class="prog-title title-2">CC Exhibition</h3>
+                    <p class="prog-details cc-details">
+                        Let's meet the creations of artists from various fields who share the open spirit of CC.
+                    </p>
+                </article>
+                <article class="prog program-3">
+                    <div class="program-icon-img">
+                        <img src="images/program_icon_03.png" alt="Forum Icon">
+                    </div>
+                    <h3 class="prog-title title-3">Forum</h3>
+                    <p class="prog-details forum-details">
+                        We have time to share our thoughts and opinions with experts by topic.
+                    </p>
+                </article>
+                <article class="prog program-4">
+                    <div class="program-icon-img">
+                        <img src="images/program_icon_04.png" alt="workshop Icon">
+                    </div>
+                    <h3 class="prog-title title-4">Workshop</h3>
+                    <p class="prog-details workshop-details">
+                        You can create your own creations using open source instead of just looking at them.
+                    </p>
+                </article>
+                <article class="prog program-5">
+                    <div class="program-icon-img">
+                        <img src="images/program_icon_05.png" alt="ignite Icon">
+                    </div>
+                    <h3 class="prog-title title-5">CC Ignite</h3>
+                    <p class="prog-details ignite-details">
+                        Create opportunities to freely network with CC personnel from around the world.
+                    </p>
+                </article>
+            </div>
+            <div class="join-bootcamp">
+                <h3 class="join">Join CC Bootcamp 2022</h3>
+            </div>
+            <a href="#" class="all-progs">SEE ALL PROGRAMS</a>
+        </section>
+
+        <section class="speakers">
+            <h3 class="featured-speaker">Featured Speakers</h3>
+            <div class="bottom-line bold"></div>
+            <div class="div">
+                <article class="speakers-details">
+
+                </article>
+                <article class="more-speakers">
+                    <p class="more">More</p>
+                    <span class="entity">&#5167;</span>
+                </article>
```

Comment:
- [ ] [optional] I am marking this change as optional because it was missed by the previous reviewer but I strongly suggest considering them, On mobile devices when users click on the more button all the remaining speakers should appear on the screen. Please apply javascript functionalities on the more button, It will make your website fully functional and professional.

Reply:
I just hope I won't repeat this with these changes as the previous reviewer does not ask for these. I am feeling as if you are blocking me to move on

Reply:
> I just hope I won't repeat this with these changes as the previous reviewer does not ask for these. I am feeling as if you are blocking me to move on

@Ibnballo1 These are optional suggestions not required changes, If you skip them the next CR will not ask you to implement them. You can skip them but I would suggest you implement them.

Reply:
Thanks... 
I will try to work on it and I hope everything goes well
@thecodechaser

Reply:
> Thanks... I will try to work on it and I hope everything goes well @thecodechaser

@Ibnballo1 If you are out of time, First work on the required changes and get approval after you get your project approved, You can work on my suggestions later.

Reply:
I will work later on the last optional change (Javascript)

Reply:
> I will work later on the last optional change (Javascript)

@Ibnballo1 Yes, you are free to do that